contracts/CryptoChampions.sol

```diff
@@ -812,4 +836,35 @@ contract CryptoChampions is ICryptoChampions, AccessControl, ERC1155, VRFConsume
         grantRole(ROLE_GAME_ADMIN, address(priceWar));
         priceWar.startGame();
     }
+
+    /// @notice Transfers in game currency tokens from one address to another
+    /// @param to The receiving address
+    /// @param amount The amount to transfer
+    function transferInGameTokens(address to, uint256 amount) external override {
+        bytes memory data;
+        safeTransferFrom(msg.sender, to, IN_GAME_CURRENCY_ID, amount, data);
+    }
+
+    /// @notice Transfers in game currency tokens from one address to another.
+    /// @param from The sending address.  Note that the sender must be authorized to transfer funds if the sender is different from the from address.
+    /// @param to The receiving address
+    /// @param amount The amount to transfer
+    function delegatedTransferInGameTokens(
+        address from,
+        address to,
+        uint256 amount
+    ) external override {
+        bytes memory data;
+        safeTransferFrom(from, to, IN_GAME_CURRENCY_ID, amount, data);
+    }
+
+    /// @notice Returns whether or not hero has reward for the round
+    /// @param heroId The id of the hero being searched for
+    function hasRoundReward(uint256 heroId) external view returns (bool) {
+        Hero memory hero = _heroes[heroId];
+        string memory roundWinningAffinity = winningAffinitiesByRound[currentRound];
+        return
```

Comment:
Would need to add another condition to make sure hero was minted during the round. Heroes are only eligible for rewards the same round they were minted.